src/VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -37,7 +37,11 @@
             "IgnoreSqlTelemetryOptions": {
                 "QueryIgnoreSubstrings": [ "[HangFire]." ]
             }
-        }
+        },
+        // Time before first registration required
+        "RegistrationDelay": "1.00:00:00",
+        // Time before registration expire
+        "RegistrationExpiration": "30.00:00:00"
```

Comment:
Maybe it is better to not include these settings into appsetttings.json by default - so the user with a license would not think about what are these strange values and the user without a license cannot change them easily without looking to code.